bylaws.md

```diff
@@ -94,6 +93,6 @@
 1. A member in Good Standing is defined as someone who:
 	- Has paid dues on time as defined by the denhac Membership Agreement.
 	- Is not actively subject to disciplinary action. 
-2. A Simple majority vote is a vote where more than 50% of the board members must vote in favor of the motion. The Chair does not cast a vote except in the case of a tie
```

Comment:
Strike: `The Chair does not cast a vote except in the case of a tie`. 
Our bylaws were not written to strictly follow Robert's Rules of Order, however this may have been introduced as an attempt to follow that. However, even in [Robert's Rules of Order 12th edition](https://yorkcountyfireschool.org/wp-content/uploads/2023/05/Roberts-Rules-of-Order-Newly-Revised-12th-Edt.pdf), 49:21 on procedures in small boards (less than twelve members) it states `If the chairman is a member, he may, without leaving the
chair, speak in informal discussions and in debate, and vote
on all questions.`  I believe it's correct to remove this entirely rather than update it to `except to create or break a tie`.